net/netcheck/netcheck.go

```diff
@@ -1116,13 +1116,20 @@ func (c *Client) checkCaptivePortal(ctx context.Context, dm *tailcfg.DERPMap, pr
 	if err != nil {
 		return false, err
 	}
+
+	chal := "tailscale " + node.HostName
```

Comment:
Assuming this is the node's hostname or something, and assuming it's not exposed in other ways, isn't this a minor privacy issue?

Reply:
Good question! In this case, `node` is the node of the DERP server, not the local machine that Tailscale is running on. So `node.HostName` would be something like `derp10.tailscale.com` here.